devel/ci/integration/greenwave/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/factory2/greenwave:prod
+MAINTAINER "Aurelien Bompard" <abompard@fedoraproject.org>
```

Comment:
just a small nitpick here, the `MAINTAINER` instruction is depreacted, instead a `maintainer` label should be used

```
LABEL maintainer="Aurelien Bompard <abompard@fedoraproject.org>"
```
That will also save one layer in the image making it a little faster to build :)